### ISSUE TITLE
Windows issues

### ISSUE THREAD

**Issue by asvetliakov** [AndyMoreland/tsunami#19]
tsunami-code 0.5.1
VScode 1.8.1
Windows 10

I have few problems which makes this plugin totally unusable on Windows:
1) When importing symbol it rewrites all existing relative paths from ```../../path/to/file``` to ```..\..\path\to\file```
2) Importing some symbol from project is resulting to absolute path name instead of relative one

**Comment by AndyMoreland** [comment]
re: 1: that's funny. I went through and replaced all of my "/" characters with "path.separator" in an effort to be windows compatible. Clearly that was a mistake in this instance.

re: 2: is there any chance the code is public? I could take a look more easily if so.

I'll address (1) tonight.

Thanks for reporting this.

**Comment by asvetliakov** [comment]
actually 2) is very simple reproducable:
create following structure:
```
 - src
   - test1.ts
   - test2.ts
  tsconfig.json
```

tsconfig.json:
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "es2015",
      "dom"
    ],
    "module": "es2015",
    "moduleResolution": "node",
    "jsx": "react",
    "sourceMap": true,
    "experimentalDecorators": true,
    "removeComments": false,
    "noEmitOnError": true,
    "noImplicitAny": true,
    "suppressImplicitAnyIndexErrors": true,
    "strictNullChecks": true,
    "typeRoots": [
      "types",
      "node_modules/@types"
    ],
    "types": [
    ],
    "baseUrl": ".",
    "paths": {
      "*": [
        "types/*"
      ]
    }
  },
  "include": [
    "src/**/*"
  ],
  "exclude": [
    "node_modules",
    "public",
    "compiled"
  ]
}
```

test1.ts
```
export function test1(a: string): boolean {
    return false;
}
```

test2.ts:
```
test1<complete here>
```

Result:
```
import { test1 } from "c:/Users/asvet/work/test/src/test1";
```

**Comment by AndyMoreland** [comment]
cool, thank you. I'll fix that too. I know what's wrong.

**Comment by asvetliakov** [comment]
Thanks. The completion popup actually displays correct relative name if it helps.
Also thanks for your plugin, your parser is pretty good from first view - works correctly and very fast for pretty large project.

**Comment by AndyMoreland** [comment]
Hey -- I've gone ahead and released tsunami-code 0.5.2. If you update your extension (and maybe restart vscode) it should work. I tested it on a windows box on a small project.

Thanks for reporting this and please feel free to report any other bugs you encounter -- there are a few issues laying around on this github from people I know in real life that I haven't gotten around to but in general I try to fix bugs within about 24 hours.

**Comment by asvetliakov** [comment]
0.5.2 works, thanks!
